src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -900,33 +829,31 @@ def _set_dserial_params(self,
         between serializing and deserializing.
 
         Args:
-            src_pool (TestPool, optional): the source pool.
-                Alternatively, this can be the pool uuid.
-            src_cont (TestContainer, optional): the source container.
-                Alternatively, this can be the container uuid.
-            dst_pool (TestPool, optional): the destination pool.
-                Alternatively, this can be the pool uuid.
-
+            src_pool (TestPool, optional): the source pool or uuid.
+            src_cont (TestContainer, optional): the source cont or uuid.
+            dst_pool (TestPool, optional): the destination pool or uuid.
         """
         # First initialize new commands
-        self.dserialize_cmd = Dserialize(self.hostlist_clients)
-        self.ddeserialize_cmd = Ddeserialize(self.hostlist_clients)
+        self.dserialize_cmd = DserializeCommand(self.hostlist_clients,
+                                                self.workdir)
+        self.ddeserialize_cmd = DdeserializeCommand(self.hostlist_clients,
+                                                    self.workdir)
 
         # Get an intermediate path for HDF5 file(s)
         tmp_path = self.new_posix_test_path(create=False,
                                             parent=self.serial_tmp_dir)
 
         # Set the source params for dserialize
         if src_pool or src_cont:
-            param = self._format_daos_path(src_pool, src_cont, None)
-            self.dserialize_cmd.set_dserialize_params(
-                src_path=param, out_path=tmp_path)
+            self.dserialize_cmd.set_params(
+                src_path=self.format_daos_path(src_pool, src_cont),
```

Comment:
(pylint-no-member) Instance of 'DataMoverTestBase' has no 'format_daos_path' member